### DISCUSSION TITLE
【灌水】六分街潮玩店对面的404live门口怎么蹲了个人啊

### DISCUSSION THREAD

**Discussion by nanamiitsugi** [share121/inter-knot#1114] [其他]
![图片](https://github.com/user-attachments/assets/bcff81ed-a00f-4b76-82cd-23ffcbf82eb1)
嘴里还一直念叨什么“你不想在这里我也不想在这里”“关系变成没关系问题是没问题”，怪吓人的，不知道是不是来开live的歌手？能不能请治安官来抓走他？

**Comment by yihuolin** [comment]
口合口合口合

**Comment by bngvhb114514** [comment]
没啥好说的，因为我想要说的前人们都说过了

**Comment by Qiling-XwX** [comment]
![2E81C5008FA676585EF9D0ABE53B2CC9](https://github.com/user-attachments/assets/a5a58d6a-aac8-48fc-8df8-2c45b1254e6e)
